src/Server/WishList.fs

```diff
@@ -34,25 +37,29 @@ let getWishListFromDB userName =
         File.ReadAllText(fi.FullName)
         |> JsonConvert.DeserializeObject<WishList>
 
+/// Save to the database
 let saveWishListToDB (wishList:WishList) =
     try
         let fi = FileInfo(getJSONFileName wishList.UserName)
         if not fi.Directory.Exists then
             fi.Directory.Create()
         File.WriteAllText(fi.FullName,JsonConvert.SerializeObject wishList)
-    with
-    | _ -> ()
+    with exn ->
+        printfn "Save failed %A" exn 
```

Comment:
```
open Suave.Logging
open Suave.Logging.Message

let logger = Log.create "FableSample"

logger.error (eventX "Save failed with exception" >> addExn exn)
```

For example. It uses the Logary Facade. You could add the Facade to this project, too if you want to.